api_tests/files/views/test_file_detail.py

```diff
@@ -71,6 +71,7 @@ def test_get_file(self):
         assert_equal(attributes['last_touched'], None)
         assert_equal(attributes['provider'], self.file.provider)
         assert_equal(attributes['size'], self.file.versions[-1].size)
+        assert_equal(attributes['current_version'], len(self.file.history))
```

Comment:
This test is a little self-referencing, which is fine to prevent regressions but not the best for verifying the behavior. Could you make sure that in the setup that the file has a non-trivial number of versions, and that whatever you do to make those versions (`for` loop, for example) is the thing that gets compared?